setup.cfg

```diff
@@ -4,6 +4,9 @@ doctest_optionflags= NORMALIZE_WHITESPACE IGNORE_EXCEPTION_DETAIL ALLOW_UNICODE
 norecursedirs = fixture .git .tox
 testpaths = mapproxy
 
+[flake8]
+ignore = E117,E121,E122,E123,E124,E125,E126,E127,E128,E129,E131,E201,E202,E203,E221,E222,E223,E225,E226,E228,E231,E241,E251,E261,E262,E265,E266,E271,E272,E301,E302,E303,E305,E306,E402,E501,E502,E701,E702,E703,E711,E712,E713,E721,E722,E731,E741,F401,F523,F812,F821,F841,F901,W503,W504,W191,W291,W292,W293,W391
```

Comment:
It would be more helpful (in the long term) if you put each code on a separate line and also add a comment for each code. This will make future changes easier (diffs).

Maybe you find a list of all codes that you can copy&paste?

Reply:
I've updated the PR to have one code per line. I indeed got the list of code by running `flake8 --statistics` and picking all the codes that yield at least one error.

I'll send an email to the mailing list to get PSC approval before I start looking into fixing each code one by one.